topics/cluster-spec.md

```diff
@@ -33,7 +33,7 @@ order to force certain keys to be stored in the same hash slot. However during
 manual resharding, multi-key operations may become unavailable for some time
 while single key operations are always available.
 
-Redis Cluster does not support multiple databases like the stand alone version
+Redis Cluster does not support multiple databases like the standalone version
```

Comment:
```suggestion
Redis Cluster does not support multiple databases like the stand-alone version
```

Reply:
> so latency figures are also the same as the single standalone Redis node case

I saw that we also use `standalone` in other places

Reply:
Ok :)